jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -259,7 +251,7 @@ and get details of the possible cause of the issue.
 
 == Release 3.0.1
 
-- `build_agent` `version` value updated, as the previous version was relying on a vulnerable version of PsExec.
```

Comment:
I am not sure here, let's leave as is for now as this item will have been checked by engineering.

Reply:
OK, have reverted to previous version.